pkg/piecestore/rpc/client/client.go

```diff
@@ -64,8 +68,10 @@ func NewPSClient(conn *grpc.ClientConn, bandwidthMsgSize int, prikey crypto.Priv
 	}
 
 	return &Client{
-		conn:             conn,
-		route:            pb.NewPieceStoreRoutesClient(conn),
+		conn:             nil,
+		route:            nil,
```

Comment:
You can just remove the above two lines. The default vaue for these fields is `nil` if you miss setting them.